cryptocurrency-realtime-trend/src/main/java/com/hazelcast/jet/demo/common/StreamTwitterP.java

```diff
@@ -94,7 +94,6 @@ public boolean isCooperative() {
         return false;
     }
 
-    @Override
```

Comment:
Inserting @cangencer's comment on email chain : Close method was recently added, so Override is correct.